### DISCUSSION TITLE
编译的时候报错

### DISCUSSION THREAD

**Discussion by codedetermination** [robomaster-oss/rmoss_core#3] [General] [answered]
--- stderr: rmoss_interfaces                         
CMake Error at /opt/ros/foxy/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:240 (list):
  list index: 1 out of range (-1, 0)
Call Stack (most recent call first):
  CMakeLists.txt:18 (rosidl_generate_interfaces)
这个是什么原因？ 连续报了很多的一样的错误

**Comment by gezp** [comment, marked as answer]
工作目录是否包含中文路径？ @codedetermination

**Comment by codedetermination** [comment]
包含了中文路径 修改后正常了 十分感谢

**Comment by codedetermination** [comment]
[virtual_image_cam-1] /home/guoyi/Documents/rm_xunlina/install/rm_cam/lib/rm_cam/virtual_image_cam: symbol lookup error: /home/guoyi/Documents/rm_xunlina/install/rm_cam/lib/rm_cam/virtual_image_cam: undefined symbol: _ZNK15image_transport9Publisher7publishERKSt10shared_ptrIKN11sensor_msgs3msg6Image_ISaIvEEEE
[ERROR] [virtual_image_cam-1]: process has died [pid 114855, exit code 127, cmd '/home/guoyi/Documents/rm_xunlina/install/rm_cam/lib/rm_cam/virtual_image_cam --ros-args --params-file /tmp/launch_params_gae71of8 --params-file /tmp/launch_params_jwujz2h_ --params-file /tmp/launch_params_smpd06i5'].

我又遇到了查不到的错误 这个貌似是加载参数出错 但是我不明白为什么要加载tmp目录下的文件，相较于ros1 ros2中文资料很难获得 我不得不再次向您请教 在这里提前表示谢意！

**Comment by gezp** [comment]
建议使用相关问题在[discussions](https://github.com/robomaster-oss/rmoss_core/discussions)中提，issue提关于项目的错误，改进，建议等内容。另外请提供更多信息，比如你使用什么样的命令。
